compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1812,6 +1823,16 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::ILOpCode opcode
             return true;
          else
             return false;
+      case OMR::vabs:
+         if (et == TR::Int8 || et == TR::Int16 || et == TR::Int32 || et == TR::Float || et == TR::Double)
+            return true;
+         else
+            return false;
+      case OMR::vsqrt:
+         if (et == TR::Float || et == TR::Double)
+            return true;
+         else
+            return false;
```

Comment:
`return (et == TR::Float || et == TR::Double);` would be simpler for these lines.
I think it can be done in a separate PR.